src/modules/endpoint/DatabaseConnector.js

```diff
@@ -271,6 +269,26 @@ class DatabaseConnector {
     await this.writeFile(path, JSON.stringify(json))
   }
 
+  isCategoriesPersisted (context: DatabaseContext): Promise<boolean> {
```

Comment:
Grammatically this should be `areCategoriesPersisted` :stuck_out_tongue:

Reply:
Yeah but I went with is so it is consitent with the other functions.